app/assets/javascripts/discourse/app/controllers/user-private-messages.js

```diff
@@ -3,26 +3,118 @@ import { action } from "@ember/object";
 import { alias, and, equal } from "@ember/object/computed";
 import discourseComputed from "discourse-common/utils/decorators";
 import { VIEW_NAME_WARNINGS } from "discourse/routes/user-private-messages-warnings";
+import I18n from "I18n";
+
+export const personalInbox = "__personal_inbox__";
```

Comment:
In JS our convention is to use `UPPER_SNAKE_CASE` for constants. ie `PERSONAL_INBOX` and `ALL_INBOX`